assessments/intake-process.md

```diff
@@ -0,0 +1,34 @@
+# Security Assessment Priorities & Pipeline Intake Process
+
+SIG-Security has a volunteer team of subject matter experts and industry professionals dedicated to helping SIG-Security members, the TOC, and the larger CNCF community maintain an understanding of the current state of security in the cloud native ecosystem and helping cloud native projects succeed. 
+
+The following process describes how projects are prioritized for security assessments.
+
+# Authority
+
+* The [Security Assessment Facilitator](../governance/roles.md#security-assessment-facilitator) maintains the Assessment queue
+* A named chair provides oversight for the Security Assessment initiative. 
+This chair can adjust queue, and is also responsible for conveying TOC 
+instructions, when needed. (Sarah Allen is the current named Chair for Security Assessments.)
```

Comment:
Should we call out @ultrasaurus  in the doc as the current chair?  should the roles held by members go somewhere else?  want to ensure we update role changes one place instead of several places

Reply:
addressed by moving who is filling what role to new section in the repo root README (and linking from here)

Reply:
LGTM